src/local-history/local-history-manager.ts

```diff
@@ -449,4 +451,21 @@ export class LocalHistoryManager {
             .update(str, 'utf8')
             .digest('hex');
     }
+
+    /**
+     * Exclude the revisions of sources which follows the glob pattern. 
+     * @param document Represent the active text document.
+     */
+    private excludeFiles(document: vscode.TextDocument): boolean {
```

Comment:
@Anasshahidd21 no one is actually calling the method. I believe you may have pushed incomplete changes?